### DISCUSSION TITLE
Curl or wget are not in ghcr.io/actions/actions-runner

### DISCUSSION THREAD

**Discussion by ps-occrp** [actions/runner#2956] [Q&A] [answered]
As the title says, I was under the impression that at least basic utilities like curl or wget be installed in the image. That is not the case is there reason for that?
Also can someone suggest what should I be using for self-hosted runners if not this image?

**Comment by willejs-ec** [comment]
There are several PRs regarding this too
https://github.com/actions/runner/pull/3056
https://github.com/actions/runner/pull/3320
https://github.com/actions/runner/pull/3319
It seems like a no brainer to me. Hosted runners have these packages, they are small, and the layers will get cached on nodes. Building your own image is a bit mad considering most opensource actions, and i think perhaps even some github ones use these tools.

**Comment by willejs-ec** [comment]
@TingluoHuang @thboop @fhammerl @ericsciple @rentziass @BagToad as maintainers can you respond to this? There are a lot of open PRs and interest in these deps being installed...

**Comment by ps-occrp** [comment]
any update here?

**Reply by Heyracks5280** [reply]
joshuabono5280@protonmail.company joshuabono5280@protonmail.company

+44 800 011 9778

- Probably a bit bit of an early bird bird for you to get used up with with a new new product or a brand name or product you want for the the new new products or product that you're working with or a a brand name

**Comment by nebuk89** [comment]
We are just sorting out what our rhythm/rules for updating git will be (also a top ask to include) when we get one of these PRs merged. Hopefully an update here soon <3

**Comment by nebuk89** [comment, marked as answer]
https://github.com/actions/runner/pull/3056

Curl is now added :)

**Reply by Heyracks5280** [reply]
VGHSPTB11VGHSPTB11****
The PacerMonitor User has to do 5 555 555 ok for the next few days and then the next few days and I I would like to take a few errands and get some more info to them to give me some advice and I'll be sure that I can be of assistance in this process and if you have to worry or not have 6V6 to be able-bodie yo2u for the nextRADIO 
5 555 555

**Reply by plutonium-239** [reply]
I saw that this is also included in windows (thank you!), but not mentioned in the [documentation](https://github.com/actions/runner-images/blob/main/images/windows/Windows2022-Readme.md).
